index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Flutter is a mobile app SDK for iOS and Android
+title: Welcome to Flutter
```

Comment:
Ah! Have we considered getting software to help with that? Was looking at [this gem](https://github.com/jekyll/jekyll-seo-tag) for help with SEO on another site.

LMK how you'd like to proceed. I believe it's the kind of thing you can only test in a limited way locally, i.e. that it doesn't break stuff... More research needed.